src/chips/nrf51822/lib.rs

```diff
@@ -1,10 +1,16 @@
 #![crate_name = "nrf51822"]
 #![crate_type = "rlib"]
 #![feature(asm,concat_idents,const_fn)]
+#![feature(core_intrinsics)]
 #![no_std]
 
 extern crate common;
 extern crate hil;
 
 mod peripheral_registers;
+mod peripheral_interrupts;
```

Comment:
Yeah, just remove this. You don't need `peripheral_interrupts` for your changes.